src/forwardTo.js

```diff
@@ -1,19 +1,16 @@
 import wrapAction from './wrapAction';
 
-/** Modified dispatch by wrapping all the outgoing actions by composition chain
+/**
+ * Returns new dispatch which automatically wraps all the actions by types chain
  *
- * @param {Function} dispatch
- * @param {...String} action composition chain
- * @return {Function} modified dispatch
+ * @param {Function} Original dispatch
+ * @param {...String} action types composition chain
+ * @return {Function} Dispatch which automatically wraps all the actions
  */
 export default (dispatch, ...types) => {
   if (types.length === 0) {
     return dispatch;
   } else {
-    if (types.some(type => ~type.toString().indexOf('.'))) {
-      throw new Error('Action type can\'t contain a dot');
-    }
-
```

Comment:
It's been moved to `wrapAction` instead, because technically it has nothing to do with forwarding.